src/app/containers/clubInfoContainer.js

```diff
@@ -0,0 +1,26 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { firebaseConnect } from 'duckbase';
+import * as ModalActions from '../actions/modalActions';
+import * as BookActions from '../actions/bookActions';
+import ClubInfo from '../components/clubInfo/clubInfoComponent';
+import * as BookClubUtils from '../utils/bookClubUtils';
+
+const mapStateToProps = (state, ownProps) => {
+  const currentUserId = state.currentUser && state.currentUser.uid;
+  const bookClubId = ownProps.match.params.clubId;
```

Comment:
Where does `.match` come from? Is it hooked up by AuthedRoute? Looks like it's somehow grabbing `:clubId` from `path='/club/:clubId'`